src/bitbucket/BitbucketAPI.ts

```diff
@@ -73,6 +76,16 @@ export class BitbucketAPI {
         landRequestId,
         landRequestStatus,
       });
+      // Retry merge on failure if numRetries is passed
+      if (options.numRetries && options.numRetries > 0) {
```

Comment:
Can add `numRetries` to the logging above or can create a new log statement within the if statement. It will be useful for debugging in future.